src/controllers/trainee/routes.ts

```diff
@@ -13,11 +13,11 @@ const traineeRouter: Router = Router();
 traineeRouter.route('/')
     .get(authMiddleWare('getUsers', 'read'), validationHandler(validation.get), traineeController.get)
     // .get(validationHandler(validation.get),traineeController.get)
-    .post(authMiddleWare('getUsers', 'write'), validationHandler(validation.create), traineeController.create)
+    .post(authMiddleWare('getUsers', 'read'), validationHandler(validation.create), traineeController.create)
     //  .post(validationHandler(validation.create),traineeController.create)
-    .put(authMiddleWare('getUsers', 'update'), validationHandler(validation.update), traineeController.update)
+    .put(authMiddleWare('getUsers', 'read'), validationHandler(validation.update), traineeController.update)
```

Comment:
for update use update instead of read and for delete use delete

Reply:
done